polynote-kernel/src/main/scala/polynote/messages/package.scala

```diff
@@ -76,6 +76,10 @@ package object messages {
     else
       str.asInstanceOf[TinyString]
 
+    def truncatePretty(str: String): TinyString = if (str.length > 255) {
```

Comment:
Curious why we need to explicitly truncate with ellipses on the backend - can't we truncate with CSS `text-overflow: ellipsis;`?

Reply:
Because it's already been truncated on the server, so there's nothing to overflow. We truncate it on the server because it could be like a 500MB string of limited utility to the client, and there's no sense sending over the wire...

Reply:
This just changes the truncation behavior to make it prettier – truncation already happens.

Reply:
Right, it just feels like adding the ellipses is a frontend sort of thing. Sorry, I forgot that `text-overflow` wouldn't work given it's are already truncated.